book/01-introduction/sections/about-version-control.asc

```diff
@@ -22,7 +22,7 @@ To deal with this issue, programmers long ago developed local VCSs that had a si
 .Local version control.
 image::images/local.png[Local version control diagram]
 
-One of the more popular VCS tools was a system called RCS, which is still distributed with many computers today.
```

Comment:
If using `most` here then `tool` is wrong; `tools` is still correct. Some suggestions:
- `One of the most popular VCS tools was `...
- `The most popular VCS tool was `...